spec/factories/health/patients.rb

```diff
@@ -5,6 +5,7 @@
     birthdate { Date.current }
     sequence(:medicaid_id)
     enrollment_start_date { Date.current }
+    current { true }
```

Comment:
I might have missed this earlier, but isn't tithe referral that should have current?  There should be many referrals per patient I think.